packages/components/src/buttons/src/EmbeddedButton.tsx

```diff
@@ -44,7 +61,7 @@ export interface EmbeddedButtonProps extends StyledComponentProps<RACButtonProps
      * The visual style of the EmbeddedButton.
      * @default "neutral"
      */
-    variant?: "neutral" | "subdued" | "progress" | "positive" | "caution" | "negative" | "option1" | "option2" | "option3" | "option4" | "option5" | "option6";
+    variant?: EmbeddedButtonVariants;
```

Comment:
This is what I was hoping we didn't have to do.. I didn't want to have orbiter's variants as options. This adds more options that may confuse the user.
I'm not sure if there's a way around this though.
Any thoughts @alexasselin008 ?

Reply:
Maybe we should do the other way around? add hopper options in orbiter?

Reply:
I did feel rly dirty doing this

Reply:
Can you show me the issue you were having in the tests?

Reply:
In Orbiter's `Tab` component, there's the Lozenge slot which represents the Tag in Hopper. 

Lozenge has 4 variants: 

```
variant?: "informative" | "warning" | "positive" | "negative";
```

"informative" | "warning" don't exist in Hopper

Reply:
Ya that makes a lot of sense. 

Do you think there is a lot of use case like this? I guess for now we could add a mapOrbiterToHopperVariants, but leave those variant out of the typings. Typings are mostly for intellisence purposes and developer documentation, and we don't want them to see that those exists. 

Once we have a better view of how many mapOrbiterToHopperVariants things we have, we'll be able to see if a pattern emerge that we could fix in orbiter directly